examples/chef/common/clusters/media-input/MediaInputManager.cpp

```diff
@@ -0,0 +1,111 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/util/config.h>
+#ifdef EMBER_AF_PLUGIN_MEDIA_INPUT_SERVER
+#include "MediaInputManager.h"
+
+using namespace std;
+using namespace chip;
+using namespace chip::app::Clusters::MediaInput;
+
+MediaInputManager::MediaInputManager()
+{
+    mCurrentInput = 1;
+
+    for (int i = 1; i < 3; ++i)
```

Comment:
Comments from other CPP apply: add constants for magic numbers, remove TODO:insert or explain what code needs to be added. This applies throughout the PR.

Reply:
Done.